Applications/WebPortal/WebApp/library/source/pages/dashboard_page/local_dashboard_model.ts

```diff
@@ -9,13 +9,15 @@ export class LocalDashboardModel extends DashboardModel {
   /** Constructs a LocalDashboardModel.
    * @param account - The account that's logged in.
    * @param roles - The account's roles.
+   * @param groups - The account's groups.
```

Comment:
Dashoard model doesn't need groups. Like identity, it's only used on the profile page.

Reply:
Okay, instead would there be any issues with passing serviceClients as an argument to makeAccountModel and making the method async so that groups can be retrieved inside the method?

Reply:
I'm not seeing why DashboardModel and AccountModel need groups as an argument.

To me, it seems like groups should be loaded when identity is. Look over all the profile models.